app/javascript/dashboard/components/widgets/conversation/ChatFilter.vue

```diff
@@ -1,27 +1,75 @@
 <template>
-  <select v-model="activeStatus" class="status--filter" @change="onTabChange()">
-    <option
-      v-for="(value, status) in $t('CHAT_LIST.CHAT_STATUS_FILTER_ITEMS')"
-      :key="status"
-      :value="status"
+  <div>
+    <woot-button
+      variant="smooth"
+      size="small"
+      color-scheme="secondary"
+      class="selector-button"
+      @click="toggleDropdown"
     >
-      {{ value['TEXT'] }}
-    </option>
-    <option value="all">
-      {{ $t('CHAT_LIST.FILTER_ALL') }}
-    </option>
-  </select>
+      {{ $t('CHAT_LIST.CHAT_STATUS_FILTER_ITEMS')[activeStatus]['TEXT'] }}
+      <fluent-icon
+        :icon="showActionsDropdown ? 'chevron-up' : 'chevron-down'"
+        class="icon"
+        size="14"
+      />
+    </woot-button>
+    <div
+      v-if="showActionsDropdown"
+      v-on-clickaway="closeDropdown"
+      class="dropdown-pane dropdown-pane--open"
+    >
+      <woot-dropdown-menu>
+        <woot-dropdown-sub-menu
+          :title="this.$t('CHAT_LIST.CHAT_STATUS_FILTER')"
```

Comment:
```suggestion
          :title="$t('CHAT_LIST.CHAT_STATUS_FILTER')"
```

Don't need the `this`